client/src/bin/greengrass-cli.bat

```diff
@@ -0,0 +1,9 @@
+@echo off
```

Comment:
Per @JamieHunter  recommendation, I will rename this file's extension to `.cmd`.